src/sql/workbench/browser/modelComponents/loadingComponent.component.ts

```diff
@@ -67,10 +67,18 @@ export default class LoadingComponent extends ComponentBase<azdata.LoadingCompon
 		this.layout();
 	}
 
+	public override layout() {
+		super.layout();
+		if (this._component) {
+			const childComponent = this.modelStore.getComponent(this._component.id);
+			childComponent?.layout();
+		}
+	}
+
 	public override setProperties(properties: { [key: string]: any; }): void {
 		const wasLoading = this.loading;
 		super.setProperties(properties);
-		if (wasLoading && !this.loading) {
+		if (wasLoading !== this.loading) {
```

Comment:
this was introduced in https://github.com/microsoft/azuredatastudio/pull/9088/files#diff-8effc957b1a67811f03a7c6e348c947d2ae372eb933d92a0a41ff9bc9054803fR69, and while working on this issue, I realized we have an issue here, it will only announce the loading completed event and miss the loading event. so fixing it to check for changes of the loading state.